src/rust/bitbox02/src/communication_mode.rs

```diff
@@ -21,3 +21,21 @@ pub fn ble_disable() {
 pub fn ble_enabled() -> bool {
     unsafe { bitbox02_sys::communication_mode_ble_enabled() }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    // Ensures a BLE-capable platform and verifies that calling `ble_disable()`
+    // makes `ble_enabled()` return false.
+    #[test]
+    fn ble_disable_disables_ble() {
+        unsafe {
+            bitbox02_sys::fake_memory_factoryreset();
```

Comment:
Use `crate::memory::fake_memory_factoryreset()`, which already exists (you'd just maybe need to adjust the guard to `#[cfg(feature = "testing")]`, not sure if cfg(test) is file-based or crate-based)